bodhi/tests/server/test_models.py

```diff
@@ -90,7 +90,7 @@ class TestQueryProperty(BaseTestCase):
 
     def test_session(self):
         """Assert the session the query property uses is from the scoped session."""
-        query = model.Package.query
+        query = model.RpmPackage.query
```

Comment:
@jeremycline I'm not sure I can recall why it needed to be changed. Presumably some test was failing at some point, but when I change it back the tests all pass so I'll set it back. Thanks for pointing that out!